src/WCSimPrimaryGeneratorAction.cc

```diff
@@ -167,6 +167,7 @@ void WCSimPrimaryGeneratorAction::GeneratePrimaries(G4Event* anEvent)
 	    vtxs[iVertex] = G4ThreeVector(atof(token[1])*cm,
 				    atof(token[2])*cm,
 				    atof(token[3])*cm);
+	    G4double VertexTime=atof(token[4]);
```

Comment:
Doesn't this miss a time unit? I think the vector files I get from super nova simulations with sntools use milliseconds.